lib/orderbook.js

```diff
@@ -137,7 +137,10 @@ _.assign(Orderbook.prototype, new function() {
       return order.id === match.maker_order_id;
     });
 
-    assert(order);
+    if (order === -1 ) {
```

Comment:
`find` must return null on record not found (see https://github.com/vadimg/js_bintrees/blob/473c12feeb91ecb5aa4c8eaf35c4143fa84ede46/dist/rbtree.js#L25), i dont know why it returns -1, but it does; and -1 doesn't have any property `size` with a `sub` method.